src/App.tsx

```diff
@@ -1,7 +1,8 @@
-import { PeoplePage } from './components/PeoplePage';
+// import { PeoplePage } from './components/PeoplePage';
```

Comment:
The `PeoplePage` component is imported but not used in this file. If it's not needed, consider removing this import to clean up the code.